src/app/games/siam/tests/SiamMinimax.spec.ts

```diff
@@ -175,28 +211,24 @@ describe('SiamMinimax:', () => {
             const node: SiamNode = new SiamNode(state, MGPOptional.empty(), MGPOptional.of(move));
             const boardValue: number = minimax.getBoardValue(node).value;
 
-            const symetryState: SiamState = new SiamState(board, 1);
-            const symmetryNode: SiamNode = new SiamNode(symetryState, MGPOptional.empty(), MGPOptional.of(move));
-            const symetryBoardValue: number = minimax.getBoardValue(symmetryNode).value;
-            expect(boardValue).withContext('Both board value should have same absolute value').toEqual(-1 * symetryBoardValue);
+            const turnOneState: SiamState = new SiamState(board, 1);
+            const turnOneNode: SiamNode = new SiamNode(turnOneState, MGPOptional.empty(), MGPOptional.of(move));
+            const turnOneBoardValue: number = minimax.getBoardValue(turnOneNode).value;
+            expect(turnOneBoardValue).withContext('Both board value should have same absolute value').toEqual(-1 * boardValue);
         });
     });
-    describe('Logical test', () => {
-        it('should get option for first turn', () => {
-            const board: Table<SiamPiece> = [
-                [_, _, _, _, _],
-                [_, _, _, _, _],
-                [_, M, M, M, _],
-                [_, _, _, _, _],
-                [_, _, _, _, _],
-            ];
-            const state: SiamState = new SiamState(board, 0);
-            const pushers: { coord: Coord, distance: number }[] =
-                SiamRules.getPushers(state, [1, 2, 3], [2]);
+    describe('pushers computations', () => {
+        it('should compute the expected pushers on first turn', () => {
+            // Given the initial state
+            const state: SiamState = SiamState.getInitialState();
+            // When computing the pushers
+            const pushers: { coord: Coord, distance: number }[] = SiamRules.getPushers(state, [1, 2, 3], [2]);
+            // Then it should ???? TODO TODO wtf is this test, the context do not correspond to what is tested!
```

Comment:
Then it should have a list of pusher should be 6 (trois vers le haut, trois vers le bas, I guess)
And those pusher should have a distance of 5 (turns to push outside the board the mountains)